gui/wxpython/gui_core/infobar.py

```diff
@@ -0,0 +1,89 @@
+
+import wx
+import wx.aui
+try:
+    import wx.lib.agw.infobar as IB
+except ImportError:
+    import wx.lib.infobar as IB
+
+
+class InfoBar(IB.InfoBar):
+    """
+    Custom Info bar
+    """
+    def __init__(self, parent):
+        IB.InfoBar.__init__(self, parent)
+
+        colBg = wx.SystemSettings.GetColour(wx.SYS_COLOUR_HIGHLIGHT)
+        self.SetBackgroundColour(colBg)
+        self.SetOwnForegroundColour(wx.SystemSettings.GetColour(wx.SYS_COLOUR_HIGHLIGHTTEXT))
+        self._text.SetBackgroundColour(colBg)
+        self._button.SetBackgroundColour(colBg)
+
+        # LAYOUT
+        sizer = wx.BoxSizer(wx.VERTICAL)
+        self.subSizerText = wx.BoxSizer(wx.HORIZONTAL)
+        self.subSizerButtons = wx.BoxSizer(wx.HORIZONTAL)
+        self.subSizerText.Add(self._icon, wx.SizerFlags().Centre().Border())
+        self.subSizerText.Add(self._text, 200, wx.ALIGN_CENTER_VERTICAL)
+        self.subSizerText.AddStretchSpacer()
+        self.subSizerText.Add(self._button, wx.SizerFlags().Centre().Border())
+        self.subSizerButtons.AddStretchSpacer()
+        sizer.Add(self.subSizerText, wx.SizerFlags().Expand())
+        sizer.Add(self.subSizerButtons, wx.SizerFlags().Expand())
+        self.SetSizer(sizer)
```

Comment:
The text is wrapped so that the X button is in the column by itself, i.e., there is less space for text causing it to wrap too early. Can you please look it too, @petrasovaa?